### ISSUE TITLE
support matrix

### ISSUE THREAD

**Issue by jquense** [react-bootstrap/dom-helpers#56]
what browsers should we actually support here? RB has a bottom at ie11, and that seems fairly standard now, should we do that? react still theoretically supports ie9+ but in practice it asks you to pollyfill things like RAF, and really only handles the inconsistencies in places where it's not much more work above the ie11 support

In a quick run through i think we'd be able to remove a fair bit of cruft doing ie11+, removing ie8 gets us a lot but the ie9-10 support is not nothing. Question is who is stuck on ie9-10? In practice those people are almost certainly on ie11. I've never had a support issue for ie9-10 specifically

**Comment by taion** [comment]
For people on IE 9 or IE 10, would the removed functionality be easily provided by the same polyfills they'd use for React?

**Comment by jquense** [comment]
Not really, apart from rAF, since most of the stuff here are essentially polyfills

**Comment by jquense** [comment]
let's remove IE8 support and see what's left. bootstrap annoyingly supports ie10 so we wouldn't get big jump in features anyway

**Comment by TrySound** [comment]
Is it ready to release?

**Comment by TrySound** [comment]
Friendly ping @jquense
